.env.development

```diff
@@ -14,6 +14,16 @@ SEGMENT_KEY=null
 SITE_NAME=Open edX
 USER_INFO_COOKIE_NAME=edx-user-info
 LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
-LOGO_TRADEMARK_URL=https://edx-cdn.org/v3/default/logo-trademark.svg
+LOGO_TRADEMARK_URL=https://rc.mitxonline.mit.edu/static/images/mit-logo.jpg
```

Comment:
not sure about committing project-specific values to this dev file.

Reply:
> not sure about committing project-specific values to this dev file.

These values are actually picked up from the parent MFE and we can leave it blank as well  but I will update it. FYI  edx logo URL  was  set as default and as these values are not picked up from here I didn't change them.

Reply:
Removed project related default envs.